### DISCUSSION TITLE
Redirect after successful sign-in

### DISCUSSION THREAD

**Discussion by sangosteve** [nextauthjs/next-auth#8197] [Help]
### Question 💬

I am using next 13.4.2. After the user is successfully authenticated, I want to check if the user is linked to an organization | workspace. If not I would to redirect the user to the org-selection route where they can create an organization and then proceed to the app. How can I achieve this:
Here is a link to my middleware gists:
https://gist.github.com/sangosteve/9a405018ed57db3e5e80064599fa54b8

### How to reproduce ☕️

https://gist.github.com/sangosteve/9a405018ed57db3e5e80064599fa54b8

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by johannbuscail** [comment]
Have you found the solution ?

**Comment by sangosteve** [comment]
No not at all.

**Comment by johannbuscail** [comment]
Any news ?

**Reply by sangosteve** [reply]
Nope!!
